packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -206,8 +207,8 @@ export function makeRumPublicApi(
 
     addAction: monitor((name: string, context?: object) => {
       addActionStrategy({
-        name,
-        context: deepClone(context as Context),
+        name: typeof name === 'string' ? name : JSON.stringify(sanitize(name)),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏any reason to this now and only for action name?

Reply:
👌Simplified to `name: sanitize(name)!`